starter-code/index.old.js

```diff
@@ -0,0 +1,22 @@
+const MongoClient = require('mongodb').MongoClient;
+
+const dbServer = 'localhost';
+const dbPort   = '27017';
+const dbName   = 'shop';
+const connectionURL = `mongodb://${dbServer}:${dbPort}/${dbName}`;
+
+// Use connect method to connect to the server
+MongoClient.connect(connectionURL, function(error, db) {
+  if(error) { console.log("Error connecting to server", error); return;}
+
+  console.log("Connected successfully to server")
+
+  // Make your queries here
+
+  db.collection('products').find().toArray((error, products) => {
+    assert.equal(error, null);
+    console.log('All Products: ', products)
+})
+
+  db.close();
+});
```

Comment:
You don't need that file; you could have deleted it. Otherwise you can keep it but nothing forces you to add it to the staging area before the commit.

Reply:
Yep it's an oversight, we renamed it to ".old" at the start and kinda forgot about it.